summary.md

```diff
@@ -1,5 +1,7 @@
-* [Assumptions](assumptions.md)
+* [Movies API example](movies/0.index.md)
```

Comment:
I'd use a root branch named `Examples` and put `Movies` under that new branch.
I'd also update `Assumptions` submenu items 1, 1.1 and 2 so these are no more under `Drafts` as it seems these are no more under that directory

Reply:
I agree with the structure. We will have more than just one API, correct?

About the drafts, indeed the files are no longer in drafts. I've been trying to find a moment to finally see if there's anything to clean up and move them to a permanent location. Hence #4

Reply:
I wanted to keep things lean. We could do this as soon as we have those other APIs. But I also don't really bother applying this now.

Reply:
> I'd also update `Assumptions` submenu items 1, 1.1 and 2 so these are no more under `Drafts` as it seems these are no more under that directory

I am not sure what you mean by that. Which items should be where? Are there broken links?